expenses/models.py

```diff
@@ -23,15 +23,15 @@ class Category(models.TextChoices):
     description = models.TextField(max_length=250, default='')
     category = models.CharField(max_length=32, choices=Category.choices, default=Category.OTHER)
     house_name = models.ForeignKey(House, on_delete=models.CASCADE)
-    amount = models.IntegerField(null=True, blank=True, validators=[MinValueValidator(0)])
+    amount = models.IntegerField(null=False, blank=False, validators=[MinValueValidator(0)])
     date = models.DateTimeField(default=timezone.now)
 
     def __str__(self):
         return f'Category:{self.category},Amount:{self.amount}'
 
     @staticmethod
-    def create_expense(house_name, amount, date, category):
-        expense = Expenses(house_name=house_name, amount=amount, date=date, category=category)
+    def create_expense(house_name, amount, date, category, description=''):
```

Comment:
Where is the test for this method?

Reply:
I added it now
![image](https://user-images.githubusercontent.com/93659937/170024583-5c9dbaf1-b829-4aff-a597-f9575cf1713e.png)

Reply:
> I added it now ![image](https://user-images.githubusercontent.com/93659937/170024583-5c9dbaf1-b829-4aff-a597-f9575cf1713e.png)

@Omeramsc

Reply:
You should first add an assertion that `expense` is not in `Expenses.objects.all()`, then add it and assert it is there (then there's no doubt it is the method that added it)

Reply:
fixed